### PR TITLE
Fix bug in format check

### PR DESCRIPTION
**Pull request by e10harvey**

### CODE REVIEW

**Review by lucbv — APPROVED** [review]
Thanks for fixing this, these are the annoying corner cases!

File: .github/workflows/format.yml [hunk 1]
```diff
@@ -22,8 +22,10 @@ jobs:
 
           # For every file changed, apply clang-format
           for file in $(git diff --name-only origin/$GITHUB_BASE_REF | egrep '.*\.cpp$|.*\.hpp$|.*\.h$'); do
-            clang-format-8 -i -style=file $file
-            git add $file
+            if [ -e $file ]; then
+              clang-format-8 -i -style=file $file
+              git add $file
```
Comment (dalg24) [inline .github/workflows/format.yml:27]: Why do you bother with `git add`?

I would have just done `git diff --exit-code` below instead of `git status`

**Review by dalg24 — APPROVED** [review]

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Failure: Timed out waiting for job KokkosKernels_PullRequest_Tpls_CUDA9_GCC720_Light_Tpls_GCC720_GCC740 to start: Total Wait = 3603
- Other jobs have been previously started - We must stop them...

**Comment by dalg24** [comment]
There is no point on running the tests on the Sandia machines for that PR

**Comment by dalg24** [comment]
Try now

**Pull request merged**